### PR TITLE
Version 0.9.1

### PR DESCRIPTION
**Pull request by FriendsOfGalaxyBot**
Sync with the original repository

### CODE REVIEW

File: wgc/wgc_application_local.py [hunk 1]
```diff
@@ -1,143 +1,78 @@
...
+        return self.__metadata.get_app_id()
 
+    def get_app_size(self) -> int:
+        total_size = 0
+        try:
+            for dirpath, _, filenames in os.walk(self.__folder):
+                for f in filenames:
+                    fp = os.path.join(dirpath, f)
+                    if not os.path.islink(fp):
+                        total_size += os.path.getsize(fp)
```
Comment (FriendsOfGalaxy) [inline wgc/wgc_application_local.py:45]: This loop performs blocking i/o operations and may take a few seconds to finish. I don't know how many files to be stat are in Wargaming games but if many (or user has slow HDD), then Galaxy may kill the plugin as it does not respond for `ping` signal.

See https://github.com/FriendsOfGalaxy/galaxy-integration-epic/issues/6

I recommend using dummy `await asyncio.sleep(0)` between loop iterations or put this into separate thread/event loop.

**Review by FriendsOfGalaxy — CHANGES_REQUESTED** [review]

File: wgc/wgc_application_local.py (hunk 1 above)
Comment (Mixaill) [reply wgc/wgc_application_local.py:45]: Fixed in https://github.com/Mixaill/galaxy-integration-wargaming/commit/5a48d554cc5b82b8e9384715626e6a2924690947

File: wgc/wgc_keccak.py [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+        Converts a sequence of byte values to a string.
+        """
+        return bytes(bl)
+
+    @staticmethod
+    def str2bytes(bs: bytes) -> list:
+        """
+        Converts a string to a sequence of byte values.
+        """
+        return list(bs)
```
Comment (FriendsOfGalaxy) [inline wgc/wgc_keccak.py]: Probably something wrong with docstrings

Comment (Mixaill) [reply wgc/wgc_keccak.py]: Nope, it is correct in this case. `bytes` represents here ASCII-encoded string

Comment (Mixaill) [reply wgc/wgc_keccak.py]: simplified in 31b6f6954b307505e25dca6a62030e14878adcfc

**Pull request merged**